File: app/src/main/java/com/example/owner_pc/animechecker/model/entity/StaffPage.java
package com.example.owner_pc.animechecker.model.entity;

import java.util.ArrayList;

/**
 * Created by owner-PC on 2017/05/21.
 */

public class StaffPage extends Staff {
    public ArrayList<Anime> animeStaff;

    @Override
    public String toString() {
        return "StaffPage{" +
                "animeStaff=" + animeStaff +
                '}';
    }
}

File: app/src/main/java/com/example/owner_pc/animechecker/model/entity/StaffPage.kt
package com.example.owner_pc.animechecker.model.entity

import java.util.ArrayList

/**
 * Created by owner-PC on 2017/05/21.
 */

class StaffPage : Staff() {
    var animeStaff: ArrayList<Anime>? = null

    override fun toString(): String {
        return "StaffPage{" +
                "animeStaff=" + animeStaff +
                '}'
    }
}
